packages/label-sync/src/labels.json

```diff
@@ -55,6 +55,11 @@
       "description": "Request for information or clarification. Not an issue.",
       "color": "c5def5"
     },
+    {
+      "name": "priority: p3",
+      "description": "",
+      "color": "ffffc7"
+    },
     {
       "name": "priority: p2",
       "description": "Lowest priority. Fix may not be included in next release.",
```

Comment:
P2 Current: Lowest priority. Fix may not be included in next release.
P2 Suggest: Moderately-important Issue. Fix may not be included in next release.

P1 Current: Medium priority. Will be fixed prior to next release.
P1 Suggest: Important issue which blocks shipping the next release. Will be fixed prior to next release.

P0 Current: Highest priority. Critical issue. P0 implies highest priority.
P0 Suggest: -